sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-shared.service.ts

```diff
@@ -671,4 +671,31 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
   private getDistinctValueCodes(awards: DisbursementValue[]): string[] {
     return [...new Set(awards.map((award) => award.valueCode))];
   }
+
+  /**
+   * Total BCSL amount that the student received from the SIMS.
+   * @param studentId student id.
+   * @returns total BCSL amount that the student received from the SIMS.
+   */
+  async totalDisbursedBCSLAmount(studentId: number): Promise<number> {
```

Comment:
As part of the upcoming ticket, we can also discuss the possibility of stopping doing the calculation every time and maybe have the SUM for the students consolidated in some other table on SIMS.